### PR TITLE
Write N, Q and move played to TF examples

### PR DESCRIPTION
**Pull request by tommadams**

### CODE REVIEW

File: cc/tf_utils.cc [hunk 1]
```diff
@@ -69,6 +69,15 @@ tensorflow::Example MakeTfExample(const Tensor<uint8_t>& features,
   // outcome is a single float.
   dst_features["outcome"].mutable_float_list()->add_value(outcome);
 
+  // Q is a single float.
+  dst_features["q"].mutable_float_list()->add_value(Q);
+
+  // Number of reads is a single float.
```
Comment (amj) [inline cc/tf_utils.cc]: why a float?

File: cc/tf_utils.cc [hunk 2]
```diff
@@ -117,7 +126,9 @@ std::vector<tensorflow::Example> MakeExamples(
     game.GetPositionHistory(i, kMaxPositionHistory, &input.position_history);
 
     feature_desc.set_bytes({&input}, &features);
-    examples.push_back(MakeTfExample(features, move->search_pi.value(), game.result()));
+    examples.push_back(MakeTfExample(
+          features, move->search_pi.value(), move->Q, move->N, move->c,
```
Comment (amj) [inline cc/tf_utils.cc]: we're probably gonna need to change our BT code too at some point before v18, right

File: cc/tf_utils.cc (hunk 1 above)
Comment (tommadams) [reply cc/tf_utils.cc]: bad comment

File: cc/tf_utils.cc (hunk 2 above)
Comment (tommadams) [reply cc/tf_utils.cc]: this will work as-is but we'll have to change the BT code if we want to store Q & N in separate columns

**Pull request merged**